### PR TITLE
16 wrap multi chart graphics

### PR DESCRIPTION
**Pull request by tumayt**
Wrapped repeated charts layout, chart and plot creation into relevant functions.
updates #16

### CODE REVIEW

**Review by aasseman — CHANGES_REQUESTED** [review]
Tested and works. Can we however re-enable Anti-Aliasing?
LGTM otherwise.

File: simulation/multi_agent_simulation.py [hunk 1]
```diff
@@ -39,149 +183,130 @@ async def main():
...
-    policy_plot_legend.setParentItem(policy_plot_vb)
-    policy_plot.setClipToView(True)
-    policy_plot.setLabel("left", "Query rate")
-    policy_plot.setLabel("bottom", "Price multiplier")
-    policy_plot.setLogMode(LOG_PLOT, False)
+    layout = create_layout(
+        WINDOW_SIZE,
+        "Multi-agent training",
+        not args.save,
+        antialias=True,
```
Comment (tumayt) [inline simulation/multi_agent_simulation.py:190]: Antialiasing enabled here

Comment (aasseman) [reply simulation/multi_agent_simulation.py:190]: Ah indeed. Tried with `False` and it indeed looks worse. Somehow the AA on my computer looked worse than I remembered :stuck_out_tongue: .

**Review by aasseman — APPROVED** [review]
LGTM!

I recommend a Squash Merge here.

**Pull request merged**